python/datasources/acs_health_insurance.py

```diff
@@ -0,0 +1,612 @@
+import json
```

Comment:
In general, the logic was a bit hard to follow for me.. could you add a multi-line comment at the top of the file describing at a high-level what this code is supposed to do & what options are available (eg local vs on gcp)